packages/backend/src/graphql/mutations/create-flow.ts

```diff
@@ -27,15 +26,15 @@ const createFlow = async (
       .throwIfNotFound()
   }
 
-  await Step.query().insert({
+  await flow.$relatedQuery('steps').insert({
     flowId: flow.id,
     type: 'trigger',
     position: 1,
     appKey,
     connectionId,
   })
 
-  await Step.query().insert({
+  await flow.$relatedQuery('steps').insert({
```

Comment:
why?

Reply:
It's cos if we start from root `Step`, the objection hooks' `asFindQuery` will return _all_ steps. Seems like footgun, especially if we are writing more hooks in the future.

I checked the generated SQL and it doesn't anything extra like an additional select, so if we do this we get hook footgun protection for free.